indico/util/redis/avatar_links.py

```diff
@@ -79,7 +81,7 @@ def init_links(user, client=None, assumeEvents=False):
     all_events = set()
     event_roles = defaultdict(set)
 
-    for link in user.linked_objects.all():
+    for link in user.linked_objects.filter(~UserLink.type.in_(['session', 'contribution'])).all():
```

Comment:
no need to worry about this - we'll completely remove the old link types eventually